generators/spa/templates/VueJS/Module.dnn

```diff
@@ -58,12 +60,12 @@
                   <moduleControl>
                     <controlKey />
                     <controlSrc>DesktopModules/<%= moduleName %>/View.html</controlSrc>
-                    <supportsPartialRendering>False</supportsPartialRendering>
+                    <supportsPartialRendering>True</supportsPartialRendering>
                     <controlTitle />
                     <controlType>View</controlType>
                     <iconFile />
                     <helpUrl />
-                    <viewOrder>0</viewOrder>
+                    <supportsPopUps>True</supportsPopUps>                    
```

Comment:
@VAlmea Why are you supporting popups here? Also, we should keep the `viewOrder`, even though it's not really used.

Reply:
@VAlmea This also was never addressed.

Reply:
Fixed! I was doing some tests guiding me through the configuration of the Page Manager module.